tools/topology/platform/intel/cml.m4

```diff
@@ -11,3 +12,75 @@ undefine(`SSP_MCLK_RATE')
 define(`SSP_MCLK_RATE', `24000000')
 
 include(`platform/intel/dmic.m4')
+
+undefine(`DMIC0_PIPE_NAME')
+define(`DMIC0_PIPE_NAME', `passthrough')
+
+ifelse(SOF_ABI_VERSION_3_9_OR_GRT, `1',
+`undefine(`SSP_PIPE_NAME')
+define(`SSP_PIPE_NAME', `volume')',
+`undefine(`SSP_PIPE_NAME')
+define(`SSP_PIPE_NAME', `src-volume')')
+
+undefine(`DMIC_PCM_NUM')
+define(`DMIC_PCM_NUM', `2')
+
+undefine(`DMIC01_FMT')
+define(`DMIC01_FMT', `s32le')
+
+undefine(`DMIC1_FMT')
+define(`DMIC1_FMT', `s24le')
+
+ifelse(SOF_ABI_VERSION_3_9_OR_GRT, `1',
+`undefine(`AMP_DAI_FMT')
+define(`AMP_DAI_FMT', `s24le')',
+`undefine(`AMP_DAI_FMT')
+define(`AMP_DAI_FMT', `s16le')')
+
+undefine(`HDMI0_INDEX')
+define(`HDMI0_INDEX', `0')
+
+undefine(`HDMI1_INDEX')
+define(`HDMI1_INDEX', `1')
+
+undefine(`HDMI2_INDEX')
+define(`HDMI2_INDEX', `2')
+
+ifelse(SOF_ABI_VERSION_3_9_OR_GRT, `1',
```

Comment:
Why do we have multiple checks for ABI version ? Sure we just need one check.

Reply:
Thanks for pointing out. Yes, one check requires only.